src/app/[locale]/login/page.tsx

```diff
@@ -1,123 +1,117 @@
-        <Image
-          className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
-          src="/next.svg"
-          alt={t('Index.body.next-js-logo')}
-          width={180}
-          height={37}
-          priority
-        />
       </div>
 
-      <div className="mb-32 grid  lg:max-w-5xl lg:w-full lg:mb-0 lg:grid-cols-4 ">
-        <a
-          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel={t('Index.body.noopener-noreferrer-0')}
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            docs{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            {t('Index.body.find-in-depth-information-about-next-js-features-and-api')}
-          </p>
-        </a>
 
-        <a
-          href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel={t('Index.body.noopener-noreferrer-1')}
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            {t('Index.body.learn')}{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            {t('Index.body.learn-about-next-js-in-an-interactive-course-with-and-nbsp-quizzes')}
-          </p>
-        </a>
+      <div className="flex flex-col items-center justify-center md:order-first md:w-1/2 ">
+        <Card className="border-none md:w-[60%]">
+          <CardHeader>
+            <p className='md:text-3xl max-sm:mt-'>Welcome <b className='md:hidden'>Back</b></p>
+            <CardTitle className='md:text-4xl max-sm:hidden '>Back</CardTitle>
+            <CardDescription className='md:hidden'>Enter your email below to create your account.</CardDescription>
+          </CardHeader>
+          <CardContent>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <div className="">
+                <Label htmlFor="email">Email</Label>
+                <Input {...register("email")} className="bg-white" id="email" placeholder="Email" />
+                {errors.email && <p>{errors.email.message?.toString() }</p>}
```

Comment:
you don't need to handle it like this see shadcn doc example 

```
"use client"

import { zodResolver } from "@hookform/resolvers/zod"
import { useForm } from "react-hook-form"
import { z } from "zod"

import { Button } from "@/components/ui/button"
import {
  Form,
  FormControl,
  FormDescription,
  FormField,
  FormItem,
  FormLabel,
  FormMessage,
} from "@/components/ui/form"
import { Input } from "@/components/ui/input"

const formSchema = z.object({
  username: z.string().min(2, {
    message: "Username must be at least 2 characters.",
  }),
})

export function ProfileForm() {
  // ...

  return (
    <Form {...form}>
      <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-8">
        <FormField
          control={form.control}
          name="username"
          render={({ field }) => (
            <FormItem>
              <FormLabel>Username</FormLabel>
              <FormControl>
                <Input placeholder="shadcn" {...field} />
              </FormControl>
              <FormDescription>
                This is your public display name.
              </FormDescription>
              <FormMessage />
            </FormItem>
          )}
        />
        <Button type="submit">Submit</Button>
      </form>
    </Form>
  )
}
```

https://ui.shadcn.com/docs/components/form